packages/components/close-button/src/close-button.tsx

```diff
@@ -48,6 +61,9 @@ export const CloseButton = forwardRef<CloseButtonProps, "button">(
       flexShrink: 0,
       ...styles,
       ...__css,
+      ...(isRounded
+        ? { borderRadius: "9999px", border: "1px solid #000" }
```

Comment:
@hirotomoyamada 
It is difficult to see the border in dark mode.
I don't think we need a border, what do you think?
![image](https://github.com/hirotomoyamada/yamada-ui/assets/9676316/8e83adbf-0494-464b-acbc-28eb71768f59)
![image](https://github.com/hirotomoyamada/yamada-ui/assets/9676316/969c4d84-0888-496c-aba0-869ad22a515a)

Reply:
Hi! @108yen @hirotomoyamada  Thanks for the review!
Maybe `currentColor` would be a good solution. What do you think?
```ts
...(isRounded
        ? { borderRadius: "9999px", border: "1px solid currentColor" }
        : {}),
```
If this solution would be good, and I'll add another commit to this PR. Do I need run `pnpm changeset` command again before committing?

![currentColor-dark](https://github.com/hirotomoyamada/yamada-ui/assets/41576384/c7e44580-92d0-46f9-92f6-4bed08e4bad9)
![currentColor-light](https://github.com/hirotomoyamada/yamada-ui/assets/41576384/61367717-60bc-465d-8dac-63e28ead2d2a)

Reply:
@108yen Thank you for the answer! I fixed the border color.